fitting_source/1D_fitting/plot_2_13C_glycine_960Hz.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-¹³C MAS NMR of Glycine (CSA) [960 Hz]
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+¹³C MAS NMR of Glycine (CSA) [960 Hz] (TO REMOVE)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Why?

Reply:
The next example is fitting all three spinning speeds. Should an example where we fit just one spectrum still exits?

Reply:
Yes. It should be fine. Just remove the `(To REMOVE)` text and let's have this example in the docs.